08-trees/TheParsingTree/testsForTheParsingTree.c

```diff
@@ -0,0 +1,43 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "theParsingTree.h"
+#include "../Tree/tree.h"
+
+bool testBuildTreeAndCalculateItFromTheTree(bool* errorCode) {
+    const FILE* test1File = fopen("test1.txt", "r");
+    if (test1File == NULL) {
+        errorCode = true;
+        return errorCode;
```

Comment:
```suggestion
        *errorCode = true;
        return true;
```
А то тут беда с типизацией. Ну, и ниже.